src/main/scala/streamit/client/FS2KafkaClient.scala

```diff
@@ -0,0 +1,66 @@
+package streamit.client
+
+import streamit._
+import cats.effect.{ ConcurrentEffect, _ }
+import fs2.Stream
+import fs2.kafka.{ Serializer => _, _ }
+import log.effect.LogWriter
+import log.effect.fs2.syntax._
+import org.apache.kafka.common.serialization.{ Deserializer, Serializer }
+import streamit.KafkaSettings
+
+object FS2KafkaClient {
+  def apply[F[_]: ConcurrentEffect: ContextShift: Timer: LogWriter](
+    settings: KafkaSettings
+  ): FS2KafkaClient[F] = new FS2KafkaClient(settings)
+}
+
+/**
+  * All fs2-kafka related activities
+  */
+class FS2KafkaClient[F[_]: ConcurrentEffect: ContextShift: Timer](settings: KafkaSettings)(
+  implicit logger: LogWriter[F]
+) {
+
+  def createConsumer[K, V](
+    topic: String
+  )(
+    implicit keyDes: Deserializer[K],
+    valDes: Deserializer[V]
+  ): Stream[F, CommittableMessage[F, K, V]] =
+    logger.infoS(
```

Comment:
if we `import cats.syntax.flatMap._` an equivalent syntax for this can be
```scala
logger.infoS(
  s"Creating consumer in ${settings.groupId} for topic $topic on server: ${settings.bootstrapServers}"
) >> consumerExecutionContextStream[F] >>= { ec =>
  consumerStream[F]
    .using(
      ConsumerSettings[K, V](keyDes, valDes, ec)
        .withAutoOffsetReset(AutoOffsetReset.Latest)
        .withClientId(KafkaClientId)
        .withBootstrapServers(settings.bootstrapServers)
        .withGroupId(settings.groupId.toString)
    )
    .evalTap(_.subscribeTo(topic))
    .flatMap(_.stream)
    .mapAsync(maxConcurrent = 25) { event =>
      ConcurrentEffect[F].pure(event)
    }
}
```

Reply:
ooh look at the fancy `>>=` :)